docs/data_management/res/cheaha_individual_account.csv

```diff
@@ -0,0 +1,6 @@
+Responsibilities,User, Storage Owner's Supervisor,Research Computing
+Sponsor external collaborator,Yes ✔️,Yes ✔️,Yes ✔️
+Create personal Cheaha account,Yes ✔️,,
+Move unused data to LTS or archive,Yes ✔️,,
+Manage backup plans,Yes ✔️,,
+Data and access control,,[With Security Exception](#security-exceptions-for-accessing-former-uab-personnel-data),
```

Comment:
Let's change this last line to match the three "Data Security Control" lines from the "Cheaha Project Directory.csv".

Administrative:
  - Individual: "Yes"
  - Supervisor: Reuse the existing link "With Security Exception"
Technical/Physical:
  - Research Computing: "Yes"